File: app/src/main/java/com/waitou/towards/bean/ThemeInfo.java
package com.waitou.towards.bean;

import android.databinding.BaseObservable;
import android.databinding.Bindable;

import com.waitou.towards.BR;
import com.waitou.wt_library.theme.ThemeEnum;

/**
 * Created by waitou on 17/2/8.
 */

public class ThemeInfo extends BaseObservable {

    public ThemeEnum themeEnum;

    @Bindable
    public boolean focus;

    public void setFocus(boolean focus) {
        this.focus = focus;
        notifyPropertyChanged(BR.focus);
    }

}

File: app/src/main/java/com/waitou/towards/bean/ThemeInfo.kt
package com.waitou.towards.bean

import android.databinding.BaseObservable
import android.databinding.Bindable
import com.to.aboomy.theme_lib.config.ThemeModule
import com.waitou.towards.BR

/**
 * Created by waitou on 17/2/8.
 */

class ThemeInfo : BaseObservable() {

    var themeModule: ThemeModule? = null

    @Bindable
    var focus: Boolean = false
        set(value) {
            //focus = value相当于set(value)
            field = value
            notifyPropertyChanged(BR.focus)
        }
}
